device-types/Ruckus/r760.yaml

```diff
@@ -0,0 +1,31 @@
+---
+manufacturer: Ruckus
+model: R760
+slug: ruckus-r760
+part_number: 901-R760-XX00
+u_height: 0.0
+is_full_depth: false
+airflow: passive
+comments: '[Ruckus R750 Datasheet](https://www.commscope.com/product-type/enterprise-networking/wireless-access-points/indoor/r760/)'
+weight: 1.76
+weight_unit: kg
+console-ports:
+  - name: USB
+    type: usb-a
+power-ports:
+  - name: DC
+    type: dc-terminal
+    maximum_draw: 38
+interfaces:
+  - name: Port 1
+    type: 1000base-t
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
```

Comment:
I highly doubt these are POE ports. Are you absolutely sure about this?

Reply:
These actually are incorrect for this access point.
line 23, should be poe_type:  type3-ieee802.3bt
per the documentation - https://www.ruckusnetworks.com/globalassets/digizuite/985089-ds-ruckus-r760.pdf  -- ( PAGE 5 - Top left, Power Consumption )